portals/devportal/src/main/java/org/wso2/carbon/apimgt/ui/devportal/Util.java

```diff
@@ -0,0 +1,202 @@
+/*
+
+
+ * Copyright (c) 2017, WSO2 LLC (http://www.wso2.org) All Rights Reserved.
```

Comment:
Should we change the date here?

Reply:
I think as per the email the year should be like this.. 

> +1. We need to update WSO2 Inc to WSO2 LLC for all releases that happened after June 14, 2022. If we have included a date range in the copyright notice then the date range should also change to (2017-2022). But if we have just used the start date in the copyright notice, then it will get updated as i.e. (c) 2017, WSO2 LLC.